hasura/metadata/databases/default/tables/public_user.yaml

```diff
@@ -34,7 +34,9 @@ select_permissions:
         - email
         - id
         - is_subscribed
+        - name
         - team_id
+        - world_id_nullifier
```

Comment:
@michaelfurutin Why do we need to let user pick this field?

Reply:
Maybe it is the presence of this hash that means the user is verified, but I am not sure.

Reply:
Ah, yeah Pretty likely 😅
@paolodamico Can you confirm this?